easybuild/easyconfigs/m/maze/maze-setup-py-python3.patch

```diff
@@ -0,0 +1,142 @@
+         print('#', cmd)
+-    mummer_out = check_output(cmd, shell=True)
++    mummer_out = check_output(cmd, shell=True).decode()
+     fwd = []
+     rev = []
+     for l in mummer_out.splitlines():
+diff -ruN maze-106803d350719188181ce39cbb5c832822b724da/maze_server.py maze-106803d350719188181ce39cbb5c832822b724da-edit/maze_server.py
+--- maze-106803d350719188181ce39cbb5c832822b724da/maze_server.py	2017-01-24 14:02:32.000000000 +0100
++++ maze-106803d350719188181ce39cbb5c832822b724da-edit/maze_server.py	2021-03-08 14:56:35.835797882 +0100
+@@ -13,10 +13,10 @@
+ import re
+ import inspect
+ 
++absPath = os.path.dirname(os.path.abspath(inspect.stack()[0][1]))
++os.chdir(absPath)
+ app = Flask(__name__)
+ cfg = {}
+-absPath = os.path.dirname(os.path.abspath(inspect.stack()[0][1]))
+-
+ 
+ @app.route('/')
+ def index():
+@@ -34,8 +35,8 @@
+     # pairwise mode:
+     if len(queries) == len(refs):
+         for ref, query in zip(refs, queries):
+-            with NamedTemporaryFile(delete=False) as f_ref, \
+-                 NamedTemporaryFile(delete=False) as f_query:
++            with NamedTemporaryFile(mode="w", delete=False) as f_ref, \
++                 NamedTemporaryFile(mode="w", delete=False) as f_query:
+                 fn_ref = f_ref.name
+                 print('>{}'.format(ref['name']), file=f_ref)
+                 print(ref['seq'], file=f_ref)
+@@ -54,12 +55,12 @@
+ 
+     # classical mode
+     else:
+-        with NamedTemporaryFile(delete=False) as f_ref:
++        with NamedTemporaryFile(mode="w", delete=False) as f_ref:
+             fn_ref = f_ref.name
+             print('>{}'.format(refs[0]['name']), file=f_ref)
+             print(refs[0]['seq'], file=f_ref)
+         for query in queries:
+-            with NamedTemporaryFile(delete=False) as f_query:
++            with NamedTemporaryFile(mode="w", delete=False) as f_query:
+                 fn_query = f_query.name
+                 print('>{}'.format(query['name']), file=f_query)
+                 print(query['seq'], file=f_query)
+diff -ruN maze-106803d350719188181ce39cbb5c832822b724da/readfq.py maze-106803d350719188181ce39cbb5c832822b724da-edit/readfq.py
+--- maze-106803d350719188181ce39cbb5c832822b724da/readfq.py	2017-01-24 14:02:32.000000000 +0100
++++ maze-106803d350719188181ce39cbb5c832822b724da-edit/readfq.py	2021-03-08 13:27:58.676966222 +0100
+@@ -38,4 +38,4 @@
+         n += 1
+         slen += len(seq)
+         qlen += qual and len(qual) or 0
+-    print n, '\t', slen, '\t', qlen
++    print(n, '\t', slen, '\t', qlen)
+diff -ruN maze-106803d350719188181ce39cbb5c832822b724da/setup.py maze-106803d350719188181ce39cbb5c832822b724da-edit/setup.py
+--- maze-106803d350719188181ce39cbb5c832822b724da/setup.py	1970-01-01 01:00:00.000000000 +0100
++++ maze-106803d350719188181ce39cbb5c832822b724da-edit/setup.py	2021-03-08 13:48:26.469241821 +0100
+@@ -0,0 +1,20 @@
++from setuptools import setup, find_packages
++
++setup(
++    name='maze',
++    version='0.0.1',
```

Comment:
Is this version used somewhere? I guess not, but maybe make this `0.0.0` and add a comment?